gqcp/include/Operator/SecondQuantized/ModelHamiltonian/HoppingMatrix.hpp

```diff
@@ -49,41 +53,41 @@ class HoppingMatrix:
      *  @param H        The Hubbard hopping matrix, represented as a `SquareMatrix`.
      */
     HoppingMatrix(const SquareMatrix<Scalar>& H) :
-        SquareMatrix<Scalar>(H) {
+        H {H} {
 
         if (!H.isHermitian()) {
             throw std::invalid_argument("HoppingMatrix::HoppingMatrix(const SquareMatrix<Scalar>&): The given hopping matrix must be Hermitian.");
         }
     }
 
 
-    /**
-     *  The default constructor.
-     */
-    HoppingMatrix() :
-        SquareMatrix<Scalar>() {}
+    // /**
+    //  *  The default constructor.
+    //  */
+    // HoppingMatrix() :
+    //     SquareMatrix<Scalar>() {}
```

Comment:
Same question here, but since you're doing it here as well, I'm assuming that it's meant to be commented out.